### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by luiskar009**

### CODE REVIEW

File: Assignment2/Luis Carlos Navas Garcia.rdf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?mso-application progid="Word.Document"?>
```
Comment (ocorcho) [inline Assignment2/Luis Carlos Navas Garcia.rdf]: This is wrong. It looks like a Word document instead

File: Assignment2/Luis Carlos Navas Garcia.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
```
Comment (ocorcho) [inline Assignment2/Luis Carlos Navas Garcia.ttl]: Again, all these lines are weird

**Comment by luiskar009** [comment]
Fixed corrupted files

File: Assignment2/Luis Carlos Navas Garcia.ttl [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-<class:Class01> ns0:includes <class:Sensor029>, <class:Computer101> .\
-<class:Sensor029> ns0:hasMeasurement <class:Measurement8401> .\
-<class:Computer101> ns0:hasOwner <class:User10A> .\
-<class:Measurement8401>\
-  ns0:hasTemperature "29" ;\
-  ns0:atTime "2010-06-12T12:00:12"^^xsd:dateTime .\
-\
-<class:User10A> ns0:hasName "Pedro" .}
\ No newline at end of file
+<class:Class01> ns0:includes <class:Sensor029>, <class:Computer101> .
```
Comment (ocorcho) [inline Assignment2/Luis Carlos Navas Garcia.ttl]: class is not defined, and it should be in that case class:Class01, and not <class:Class01>. The same applies in many other lines.

File: Assignment2/Luis Carlos Navas Garcia.ttl [hunk 4]
```diff
@@ -1,18 +1,11 @@
...
+<class:Sensor029> ns0:hasMeasurement <class:Measurement8401> .
+<class:Computer101> ns0:hasOwner <class:User10A> .
+<class:Measurement8401>
+  ns0:hasTemperature "29" ;
```
Comment (ocorcho) [inline Assignment2/Luis Carlos Navas Garcia.ttl]: This should be typed as an integer

**Comment by ocorcho** [comment]
Changes have not been done, so I am closing the pull request.

**Pull request closed without merging**